tests/utilities/mod.rs

```diff
@@ -11,6 +11,7 @@ pub trait Verify: SrcCode {
 
     fn generate_and_verify(&self) -> String {
         let src_code = self.generate();
+        println!("{}", &src_code);
```

Comment:
Can your remove the println(src_code) from the other test that use generate_and_verify?